src/test/kotlin/calculator/StringCalculatorTest.kt

```diff
@@ -0,0 +1,94 @@
+package calculator
+
+import calculator.exception.DivideByZeroException
+import calculator.exception.IllegalExpressionException
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatExceptionOfType
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+
+class StringCalculatorTest {
+    @Test
+    fun `기본 예제 테스트`() {
+        val expression = "2 + 3 * 4 / 2"
+        val result = StringCalculator.calculate(expression)
+
+        assertThat(result).isEqualTo(10F)
+    }
+
+    @Test
+    fun `기본 덧셈 테스트`() {
+        val result = StringCalculator.add(1F, 12F)
+
+        assertThat(result).isEqualTo(13F)
+    }
+
+    @ParameterizedTest
+    @CsvSource("1F,12F", "12F,1F")
+    fun `기본 뺄셈 테스트`(num1: Float, num2: Float) {
+        val result1 = StringCalculator.subtract(1F, 12F)
+        val result2 = StringCalculator.subtract(12F, 1F)
+
+        assertThat(result1).isEqualTo(-11F)
+        assertThat(result2).isEqualTo(11F)
+    }
```

Comment:
`@ParameterizedTest` 💯 
아래와 같이 테스트해보는건 어떨까요?
```suggestion
    @ParameterizedTest
    @CsvSource("1F,12F,-11F", "12F,1F,11F")
    fun `기본 뺄셈 테스트`(num1: Float, num2: Float, expect: Float) {
        val result = StringCalculator.subtract(1F, 12F)

        assertThat(result).isEqualTo(expect)
    }
```